packages/react-charting/src/components/CommonComponents/CartesianChart.styles.ts

```diff
@@ -15,13 +15,22 @@ export const getStyles = (props: ICartesianChartStyleProps): ICartesianChartStyl
         width: '100%',
         height: '100%',
         flexDirection: 'column',
-        overflow: 'hidden',
       },
       className,
     ],
     chartWrapper: {
       overflow: 'auto',
     },
+    tooltipTitle: {
+      textAlign: 'center',
+      fontFamily: 'Segoe UI',
```

Comment:
fontFamily is inherited from theme. Need not define here

Reply:
done